paper-tab.html

```diff
@@ -36,6 +37,7 @@
 `--paper-tab` | Mixin applied to the tab | `{}`
 `--paper-tab-content` | Mixin applied to the tab content | `{}`
 `--paper-tab-content-unselected` | Mixin applied to the tab content when the tab is not selected | `{}`
+`--paper-font-common-base` | Font mixin | Defined in `paper-styles/typography.html`
```

Comment:
I don't think we should advertise this mixin since it's not intended to be overwritten; `--paper-tab` should cover that case.